### ISSUE TITLE
Ignoring @typicalname on static members/methods

### ISSUE THREAD

**Issue by Melatonin64** [jsdoc2md/jsdoc-to-markdown#114]
Howdy!

`@typicalname` is a very useful feature, but I was wondering if there's a way to define a `@typicalname` for a class, and ignore it for static members/methods.
Currently, the `@typicalname` will be used also for static members.

For example:
```js
/**
 * @class MyClass
 * @typicalname instance
 */
function MyClass() {
    /**
     * @method MyClass#instanceMethod
     */
    this.instanceMethod = function() {
        console.log('I wanna document this as: instance.instanceMethod()');
    };
}

/**
 * @method MyClass.staticMethod
 */
MyClass.staticMethod = function() {
    console.log('I wanna document this as: MyClass.staticMethod()');
};
```

**Comment by 75lb** [comment]
just to let you know, i have reproduced this and the fix is coming.

**Comment by Melatonin64** [comment]
Thanks!
I managed to work around it for now by overriding partials/helpers (added a custom `@typicalinstancename` functionality)...

**Comment by 75lb** [comment]
Fixed and released in dmd@3.0.1, please update jsdoc2md.

**Comment by Melatonin64** [comment]
Thanks, can confirm it's fixed.
